priv/perf/apps/load_test/lib/runner/standard_exits.ex

```diff
@@ -0,0 +1,60 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule LoadTest.Runner.StandardExits do
+  @moduledoc """
+  Runs the Standard Exit load test scenarios.
+
+  Run with `mix test apps/load_test/test/load_tests/runner/standard_exit_test.exs`
+
+  The ManyStandardExits scenarios first creates and funds a new address, then
+  creates many utxos and then starts an exits on each one. It then waits for the
+  Watcher to sync with the root chain.
+
+  Finally, it calls Watcher status.get to measure the timing.
+
+  """
+  use Chaperon.LoadTest
+
+  alias ExPlasma.Encoding
+  alias LoadTest.ChildChain.Exit
+  alias LoadTest.Service.Faucet
+
+  @default_config %{
+    concurrent_sessions: 1,
+    exits_per_session: 1
+  }
+
+  def default_config() do
+    Application.get_env(:load_test, :standard_exit_test_config, @default_config)
+  end
+
+  def scenarios() do
+    test_currency = Application.fetch_env!(:load_test, :test_currency)
+    gas_price = Application.fetch_env!(:load_test, :gas_price)
+    config = default_config()
+
+    # Use the faucet account to add the token's exit queue if necessary
+    {:ok, faucet} = Faucet.get_faucet()
+    _ = Exit.add_exit_queue(1, Encoding.to_binary(test_currency), faucet, gas_price)
+
+    [
+      {{config.concurrent_sessions, [LoadTest.Scenario.ManyStandardExits, LoadTest.Scenario.WatcherStatus]},
```

Comment:
just curious, what is the purpose of watcher status call at the end?

Reply:
That was in the original test - start many standard exits and measure the response of watcher status.get

Reply:
I guess if we cannot understand that or justify that ..... we can probably throw it away?

Reply:
I don't think it's a bad idea to have it. 
Starting exits happens mainly on the root chain (apart from getting the exit data), so we want to have some indication of how the watcher / childchain is behaving as a result.

Can you think of a better way of measuring that other than status.get?

Reply:
You mean....testing watcher is still alive?

Reply:
Well, testing that it's still alive is a start and we can probably do that manually when testing the limits of the system.

But long term, I mean testing how long it takes to respond to status.get and tracking if any code changes affect that.
Otherwise we're just executing root chain transactions with no visibility on how that affects our services.

Reply:
sure. I was more like just wondering what is that single call to watcher status is doing. If we know currently it help sanity checks watcher is still alive after the load, I am okay to keep it as long as we know what that is doing now.